### PR TITLE
std::tuple conversion

### PR DESCRIPTION
**Pull request by neheb**

### CODE REVIEW

File: src/file_request_handler.cc [hunk 1]
```diff
@@ -73,7 +75,7 @@ static bool checkFileAndSubtitle(fs::path& path, const std::shared_ptr<CdsObject
         }
         throw_std_runtime_error("Failed to open {}: {}", path.c_str(), std::strerror(errno));
     }
-    return is_srt;
+    return { is_srt, mimeType, statbuf };
```
Comment (KarlStraussberger) [inline src/file_request_handler.cc:78]: That copies the whole struct which results in a similar issue as https://github.com/gerbera/gerbera/security/code-scanning/398?query=ref%3Arefs%2Fheads%2Fmaster

Comment (neheb) [reply src/file_request_handler.cc:78]: dead link for me.

Comment (KarlStraussberger) [reply src/file_request_handler.cc:78]: grafik

Comment (neheb) [reply src/file_request_handler.cc:78]: Out of curiosity, does that check trigger elsewhere?

Comment (KarlStraussberger) [reply src/file_request_handler.cc:78]: > Out of curiosity, does that check trigger elsewhere?

CodeQL triggered at the new struct which is in the screenshot. Nowhere else, right now.

Comment (neheb) [reply src/file_request_handler.cc:78]: Strange that this PR didn't touch that code.

**Pull request closed without merging**